plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -180,11 +181,13 @@ public void onEnable() {
             return;
         }
 
-        this.rankManager = new RankManager(this.pluginConfiguration);
         this.userManager = new UserManager();
         this.guildManager = new GuildManager(this.pluginConfiguration);
+        this.rankManager = new RankManager(this.pluginConfiguration);
         this.regionManager = new RegionManager(this.pluginConfiguration);
 
+        new TopFactory(pluginConfiguration, rankManager).addDefaultTops(userManager, guildManager);
```

Comment:
Robienie takich obiektów, które sobie wiszą i do niczego nie należą jest średnie - szczególnie z tymi nazwami. Faktorka, to Ci raczej tworzy jakiś obiekt i go wypluwa na wyjściu. Jak chcesz żeby to coś ustawiało w rankingu, to bym jakoś inaczej to przedstawił. Może jakieś 

```
rankManager.register(DefaultTop(userManager);
rankManager.register(DefaultTop(guildManager);
```